modular_skyrat/modules/death_consequences_perk/death_consequences_customization.dm

```diff
@@ -1,23 +1,24 @@
-/datum/preference/toggle/expand_death_consequences_options // simple filler option that lets you expand all the below
-	savefile_key = "dc_expand_options"
+/// List of all death consequence preference typepaths.
+GLOBAL_LIST_INIT_TYPED(death_consequences_prefs, /datum/preference, generate_death_consequences_prefs_list())
 
-	category = PREFERENCE_CATEGORY_NON_CONTEXTUAL
-	savefile_identifier = PREFERENCE_CHARACTER
+/proc/generate_death_consequences_prefs_list()
+	RETURN_TYPE(/list/datum/preference)
 
-	default_value = FALSE
+	var/list/datum/preference/pref_list = list()
 
-/datum/preference/toggle/expand_death_consequences_options/is_accessible(datum/preferences/preferences)
-	. = ..()
-	if (!.)
-		return FALSE
+	for (var/datum/preference/possible_dc_pref_type as anything in (subtypesof(/datum/preference/numeric/death_consequences) + subtypesof(/datum/preference/toggle/death_consequences)))
+		if (initial(possible_dc_pref_type.abstract_type) == possible_dc_pref_type)
+			continue
+		pref_list += possible_dc_pref_type
 
-	return DEATH_CONSEQUENCES_QUIRK_NAME in preferences.all_quirks
+	return pref_list
 
-/datum/preference/toggle/expand_death_consequences_options/apply_to_human(mob/living/carbon/human/target, value, datum/preferences/preferences)
-	return FALSE
-
-/datum/preference/toggle/death_consequences/apply_to_human(mob/living/carbon/human/target, value, datum/preferences/preferences)
-	return FALSE
+// god i fucking hate this but the only other option is making a ton of TSX types, a global list, or typechecking constantly and slapping var/name on EVERY DC pref
+/datum/preference
```

Comment:
This should be in a modular file where it makes sense, I'm *almost* certain we already have a file with `/datum/preference` in it somewhere in the master files. Let's try to keep the amount of places we add attributes to it as small as it can be, for the sake of ease of reference.